perceval/backends/core/confluence.py

```diff
@@ -331,7 +338,7 @@ def contents(self, from_date=DEFAULT_DATETIME,
         :param offset: fetch the contents starting from this offset
         :param limit: maximum number of contents to fetch per request
         """
-        resource = self.RCONTENTS + '/' + self.MSEARCH
+        resource = resource = self.RCONTENTS + '/' + self.MSEARCH
```

Comment:
?

Reply:
uh, weird, sorry about that. just pushed the fix